collect-settings.rst

```diff
@@ -3,6 +3,8 @@ Collect Settings and Security
 
 Settings are accessible from the :guilabel:`⋮` menu.
 
+There are two kings of setting, **General Settings** and **Admin Settings**. As the names suggest, this is really helpful when the person who is collecting the data and the owner of the device are not same. So the admin can set an admin password and choose what all general settings the enumerator (in this case, who is not the owner) can access. This feature can also hide/show the items displayed in the main menu and the options available in the options menu while filling the form. 
```

Comment:
This is a good addition. Can you do a self edit for brevity and clarity.

 - Avoid filler: "As the name suggests"
 - Avoid qualifications and value judgements like "really helpful"
 - Generally, avoid beginning a sentence with "So...."
 - s/"what all"/"which"

Aim for clarity, and brevity in service of clarity.